### ISSUE TITLE
Negative entropy loss, policy loss, combined loss, and explained variance for PPO

### ISSUE THREAD

**Issue by DavidChiumera-404** [DLR-RM/stable-baselines3#602]
### Question
I am training a PPO algorithm using a custom environment made for stock trading. I am using the tensorboard logging to track my training, however I am seeing a negative entropy, policy and combined loss as well as a negative explained variance. Why would this be the case?

### Additional context
From my understanding we are trying to minimize the loss to zero from some starting positive number as is found in the rest of deep learning. I tried to look up this phenomenon but couldn't find anything which explains how DRL is different from DL in that the loss functions are written in a way which allow for positive values. I have read through all the documentation as well as inspected the source. I can see from the source how there may be the potential for a negative entropy loss given the definition in the code, however this goes against my current understanding of how loss functions are supposed to be written. Any information on this would be greatly appreciated. I have included snaps of the graphs in which I am referring to. 

### Checklist
- [X] I have read the [documentation](https://stable-baselines3.readthedocs.io/en/master/) (**required**)
- [X] I have checked that there is no similar [issue](https://github.com/DLR-RM/stable-baselines3/issues) in the repo (**required**)
- 
![image](https://user-images.githubusercontent.com/82409258/136633032-54122f80-2925-4256-af44-b42f56cb6913.png)
![image](https://user-images.githubusercontent.com/82409258/136633056-04d6b2e7-6316-47f7-a172-b3d8d5fe0520.png)
![image](https://user-images.githubusercontent.com/82409258/136633082-52ac9077-8337-4731-a029-1a78b0605a4c.png)
![image](https://user-images.githubusercontent.com/82409258/136633149-efad9cdb-f840-46db-8cd7-55ef9455b4b4.png)
